eng/Versions.props

```diff
@@ -93,7 +93,9 @@
     <MicrosoftDotNetBuildTasksArchivesVersion>8.0.0-beta.23265.1</MicrosoftDotNetBuildTasksArchivesVersion>
     <MicrosoftDotNetBuildTasksInstallersVersion>8.0.0-beta.23265.1</MicrosoftDotNetBuildTasksInstallersVersion>
     <MicrosoftDotNetBuildTasksPackagingVersion>8.0.0-beta.23265.1</MicrosoftDotNetBuildTasksPackagingVersion>
-    <MicrosoftDotNetBuildTasksTargetFrameworkVersion>8.0.0-beta.23265.1</MicrosoftDotNetBuildTasksTargetFrameworkVersion>
+    <!-- Will be removed once latest SBRP flows into Arcade -->
```

Comment:
You are adding the SBRP dependency - Can't you update the reference now?

Reply:
I am just building the repo locally first to verify the change, will push the fix for it right after

Reply:
Removed